common/etc/MailScanner/MailScanner.conf

```diff
@@ -2057,6 +2057,11 @@ Spam Lists To Reach High Score = 3
 # that this (in seconds), the check is abandoned and the timeout noted.
 Spam List Timeout = 10
 
+# (yes/no) If an user sends a mails after authenticating to the local mta
+# this option disables the rbl checks if set to "yes". If set to "no" or not
+# defined the rbl check will be executed even when the user is authenticated.
+Spam List Skip If Authenticated = no
```

Comment:
Need to mention as a postfix only feature

Reply:
done